2022-03-28 11:17:39,762 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/.config/wandb/settings
2022-03-28 11:17:39,762 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/settings
2022-03-28 11:17:39,762 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-28 11:17:39,762 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-28 11:17:39,763 INFO    MainThread:12299 [wandb_init.py:_log_setup():405] Logging user logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220328_111739-p1thqtpq/logs/debug.log
2022-03-28 11:17:39,763 INFO    MainThread:12299 [wandb_init.py:_log_setup():406] Logging internal logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220328_111739-p1thqtpq/logs/debug-internal.log
2022-03-28 11:17:39,763 INFO    MainThread:12299 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f83a278a760>
2022-03-28 11:17:39,763 INFO    MainThread:12299 [wandb_init.py:init():439] calling init triggers
2022-03-28 11:17:39,763 INFO    MainThread:12299 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-28 11:17:39,763 INFO    MainThread:12299 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 143y6exh
2022-03-28 11:17:39,765 INFO    MainThread:12299 [wandb_run.py:finish():1539] finishing run afischer/tcn_samp_en-project/143y6exh
2022-03-28 11:17:39,765 INFO    MainThread:12299 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-28 11:17:39,766 INFO    MainThread:12299 [wandb_run.py:_restore():1674] restore
2022-03-28 11:17:40,443 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-28 11:17:41,080 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-28 11:17:41,200 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 120279
}

2022-03-28 11:17:41,308 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 120279
}

2022-03-28 11:17:41,414 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 120279
}

2022-03-28 11:17:41,529 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 120279
}

2022-03-28 11:17:41,636 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 120279
  total_bytes: 120279
}

2022-03-28 11:17:41,742 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 120279
  total_bytes: 120279
}

2022-03-28 11:17:41,850 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 120279
  total_bytes: 120279
}

2022-03-28 11:17:41,961 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 120279
  total_bytes: 120279
}

2022-03-28 11:17:42,065 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 120279
  total_bytes: 120279
}

2022-03-28 11:17:42,172 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 120279
  total_bytes: 120279
}

2022-03-28 11:17:42,279 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 120279
  total_bytes: 120279
}

2022-03-28 11:17:43,282 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 120279
  total_bytes: 120279
}

2022-03-28 11:17:43,754 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 120279
  total_bytes: 120279
}
local_info {
}

2022-03-28 11:17:44,881 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-28 11:17:44,882 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-28 11:17:44,885 INFO    MainThread:12299 [wandb_run.py:_footer_sync_info():2822] logging synced files
2022-03-28 11:17:44,887 INFO    MainThread:12299 [wandb_init.py:init():492] starting backend
2022-03-28 11:17:44,887 INFO    MainThread:12299 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-03-28 11:17:44,890 INFO    MainThread:12299 [backend.py:ensure_launched():219] starting backend process...
2022-03-28 11:17:44,912 INFO    MainThread:12299 [backend.py:ensure_launched():224] started backend process with pid: 14847
2022-03-28 11:17:44,914 INFO    MainThread:12299 [wandb_init.py:init():501] backend started and connected
2022-03-28 11:17:44,937 INFO    MainThread:12299 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-03-28 11:17:44,937 INFO    MainThread:12299 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-28 11:17:44,937 INFO    MainThread:12299 [wandb_init.py:init():565] updated telemetry
2022-03-28 11:17:44,976 INFO    MainThread:12299 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-28 11:17:46,092 INFO    MainThread:12299 [wandb_run.py:_on_init():1759] communicating current version
2022-03-28 11:17:46,130 INFO    MainThread:12299 [wandb_run.py:_on_init():1763] got version response 
2022-03-28 11:17:46,131 INFO    MainThread:12299 [wandb_init.py:init():625] starting run threads in backend
2022-03-28 11:17:48,415 INFO    MainThread:12299 [wandb_run.py:_console_start():1733] atexit reg
2022-03-28 11:17:48,416 INFO    MainThread:12299 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-03-28 11:17:48,416 INFO    MainThread:12299 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-03-28 11:17:48,417 INFO    MainThread:12299 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-28 11:17:48,417 INFO    MainThread:12299 [wandb_init.py:init():664] run started, returning control to user process
2022-03-28 11:17:48,417 INFO    MainThread:12299 [wandb_watch.py:watch():47] Watching
2022-03-28 11:19:55,149 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set batch_size = 30 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83b639f7f0>>
2022-03-28 11:19:55,149 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb batch_size 30 None
2022-03-28 11:19:55,149 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set epochs = 35 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83b639f7f0>>
2022-03-28 11:19:55,150 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb epochs 35 None
2022-03-28 11:19:55,150 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set lr = 0.009379951388788891 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83b639f7f0>>
2022-03-28 11:19:55,150 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb lr 0.009379951388788891 None
2022-03-28 11:19:55,150 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set no_cuda = False - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83b639f7f0>>
2022-03-28 11:19:55,150 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb no_cuda False None
2022-03-28 11:19:55,150 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set seed = 42 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83b639f7f0>>
2022-03-28 11:19:55,150 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb seed 42 None
2022-03-28 11:20:17,826 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/.config/wandb/settings
2022-03-28 11:20:17,826 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/settings
2022-03-28 11:20:17,826 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-28 11:20:17,826 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-28 11:20:17,826 INFO    MainThread:12299 [wandb_init.py:_log_setup():405] Logging user logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220328_112017-wywjpkhd/logs/debug.log
2022-03-28 11:20:17,826 INFO    MainThread:12299 [wandb_init.py:_log_setup():406] Logging internal logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220328_112017-wywjpkhd/logs/debug-internal.log
2022-03-28 11:20:17,826 INFO    MainThread:12299 [wandb_init.py:init():439] calling init triggers
2022-03-28 11:20:17,826 INFO    MainThread:12299 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-28 11:20:17,826 INFO    MainThread:12299 [wandb_init.py:init():458] re-initializing run, found existing run on stack: p1thqtpq
2022-03-28 11:20:17,828 INFO    MainThread:12299 [wandb_run.py:finish():1539] finishing run afischer/tcn_samp_en-project/p1thqtpq
2022-03-28 11:20:17,828 INFO    MainThread:12299 [jupyter.py:save_history():429] not saving jupyter history
2022-03-28 11:20:17,829 INFO    MainThread:12299 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-28 11:20:17,829 INFO    MainThread:12299 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-03-28 11:20:17,829 INFO    MainThread:12299 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-28 11:20:17,830 INFO    MainThread:12299 [wandb_run.py:_restore():1674] restore
2022-03-28 11:20:18,700 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-28 11:20:18,867 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-28 11:20:18,976 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 143649
}

2022-03-28 11:20:19,085 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 143649
}

2022-03-28 11:20:19,190 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143649
  total_bytes: 143649
}

2022-03-28 11:20:19,309 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143649
  total_bytes: 143649
}

2022-03-28 11:20:19,415 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143649
  total_bytes: 143649
}

2022-03-28 11:20:19,519 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143649
  total_bytes: 143649
}

2022-03-28 11:20:19,623 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143649
  total_bytes: 143649
}

2022-03-28 11:20:19,732 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143649
  total_bytes: 143649
}

2022-03-28 11:20:19,842 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143649
  total_bytes: 143649
}

2022-03-28 11:20:20,677 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143649
  total_bytes: 143649
}

2022-03-28 11:20:21,133 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 143649
  total_bytes: 143649
}
local_info {
}

2022-03-28 11:20:22,314 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-28 11:20:22,316 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-28 11:20:22,318 INFO    MainThread:12299 [wandb_run.py:_footer_sync_info():2822] logging synced files
